client/my-sites/plans/main.jsx

```diff
@@ -95,14 +87,8 @@ const Plans = React.createClass( {
 } );
 
 export default connect(
-	( state ) => {
-		const selectedSiteId = getSelectedSiteId( state );
-		const isPlaceholder = ! selectedSiteId;
-		return {
-			isPlaceholder,
-			plans: getPlans( state ),
-			selectedSite: getSelectedSite( state ),
-			selectedSiteId: selectedSiteId
-		};
-	}
+	( state ) => ( {
+		isPlaceholder: ! getSelectedSiteId( state ),
+		selectedSite: getSelectedSite( state )
```

Comment:
Good idea. I've pushed this a little further and removed `isPlaceholder` from the set of computed props in `connect` altogether, replacing the simple check inside the component's `render()` method instead.